src/gluonts/nursery/autogluon_tabular/estimator.py

```diff
@@ -80,9 +88,13 @@ def __init__(
     ) -> None:
         super().__init__()
 
-        self.task = task
+        self.task = tabular.TabularPredictor
```

Comment:
Can you just get rid of this line and use `tabular.TabularPredictor` in place of `self.task` below?
I don't think `self.task` is used anywhere else, and this seems like a confusing attribute to have.